### DISCUSSION TITLE
proportion table

### DISCUSSION THREAD

**Discussion by terooatt** [theislab/scvelo#725] [Q&A]
how can I get the proportion table value that is used to draw the plots?

**Comment by WeilerP** [comment]
@terooatt, I guess `scvelo.core.show_proportions` is what you are looking for.

**Comment by terooatt** [comment]
Hi Philip,

thanks for your email. I get the following error:

#####
from anndata import AnnData
from scvelo.core import show_proportions as _show_proportions

---------------------------------------------------------------------------ModuleNotFoundError
                      Traceback (most recent call
last)<ipython-input-37-8467b1685ed1> in <module>      7       8 from
anndata import AnnData----> 9 from scvelo.core import show_proportions
as _show_proportions
ModuleNotFoundError: No module named 'scvelo.core'


###

print(scvelo.__version__)

0.2.3

**Reply by WeilerP** [reply]
@terooatt, you're using an old version of scvelo. Please update to the newest one or use `scvelo.pp.show_proportions`. For future references: Please specify the package versions you are using. You can easily do so by running `scvelo.logging.print_versions()`.

**Comment by terooatt** [comment]
Basically, I'd like to get the proportion values for  splice and unspliced
read for this  data I generated which includes 120 samples. [image: Screen
Shot 2021-10-07 at 11.05.27.png]

**Comment by terooatt** [comment]
Got it, thanks!
